aws/logs_monitoring/lambda_function.py

```diff
@@ -374,15 +387,18 @@ def scrub(self, payload):
 
 def datadog_forwarder(event, context):
     """The actual lambda function entry point"""
+    global trace_connection
```

Comment:
The global comes from the layer? Why having a global instead of creating the connection here?

Reply:
We could probably move this down to `forward_traces` ?